### ISSUE TITLE
Link not working

### ISSUE THREAD

**Issue by Sharu2501** [MrShadowDev/Free-Canva-Pro-Lifetime#6]
Hi there, I got the link thank you so much but it says that the invitation has expired can you please help? Thanks !

**Comment by MrShadowDev** [comment]
Yes man it snot working I need to fix it ill do as soon as i can bro

**Comment by MrShadowDev** [comment]
Its working now pls check it again :)

**Comment by Sharu2501** [comment]
Hey! It's still not working :(

**Comment by MrShadowDev** [comment]
Yes ill need to fix it again it worked for 1 day

**Comment by Sharu2501** [comment]
oh okay thanks for the reply, will wait then. Have a good day!!
